lease/lessor.go

```diff
@@ -475,6 +476,9 @@ func (le *lessor) initAndRecover() {
 		if lpb.TTL < le.minLeaseTTL {
 			lpb.TTL = le.minLeaseTTL
 		}
+		// randomize +/-10%, so that recovered leases with same TTL
+		// do not expire all at the same time
+		lpb.TTL += rand.Int63n(lpb.TTL/10) * int64((-(i & 1))|1)
```

Comment:
`lpb.TTL *= 1.0+(0.1*(rand.Float64()-0.5))`? I think Int63n will panic if TTL < 10 since the docs say it expects >= 1.